strictEqual.js

```diff
@@ -0,0 +1,5 @@
+const sEqual = (val1, val2) => {
```

Comment:
We don't need to conserve the number of bytes our code takes up. Go ahead and call this `strictEqual` for clarity sake 😄